brave/brave-core
# fix(wallet): Missing Locale Strings Token Details

## Description 
Added missing localized strings in the `TokenDetailsModal` component.


Resolves <https://github.com/brave/brave-browser/issues/27890>

## Submitter Checklist:


## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on

## Test Plan:

1. Go to the `Portfolio` page and select a `erc-20` token.
2. Open the `More` menu and click on `Token Details`
3. If translations are available the `Labels` should translate.
